daemon/oci_windows.go

```diff
@@ -25,11 +25,30 @@ func (daemon *Daemon) createSpec(c *container.Container) (*specs.Spec, error) {
 	// In base spec
 	s.Hostname = c.FullHostname()
 
+	if err := daemon.setupSecretDir(c); err != nil {
```

Comment:
Good catch; the mount in `setupSecretDir` is actually not required; I will remove it.